dotnet/Capstone/Controllers/BirdNoteController.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Capstone.DAO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System;
+using Capstone.Models;
+using Capstone.Exceptions;
+using System.Security.Principal;
+
+namespace Capstone.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BirdNoteController : ControllerBase
+    {
+        private readonly IBirdNoteDao birdSightingDao;
+        public BirdNoteController(IBirdNoteDao birdSightingDao)
+        {
+            this.birdSightingDao = birdSightingDao;
+
+        }
+
+        [HttpGet("/bird/{birdId}/notes")]
+        public List<BirdNote> getNotes(int birdId)
+        {
+            List<BirdNote> sightingList = birdSightingDao.getSightings(birdId);
+
+            if (sightingList == null)
+            {
+                Console.WriteLine("No bird");
+            }
+
+            return sightingList;
+
+        }
+
+        [HttpGet("/note")]
+        public BirdNote getNote(int id)
+        {
+            BirdNote sighting = birdSightingDao.getBirdSighting(id);
+
+            if (sighting == null)
+            {
+                Console.WriteLine("No bird");
+            }
+
+            return sighting;
+
+        }
+
+        [HttpPost("/newNote/{birdId}")]
```

Comment:
I thought we removed this id in the path? It needs removed to match BirdNoteService.js

Reply:
Done